manifests/server/foreman.pp

```diff
@@ -0,0 +1,33 @@
+# @summary Set up Foreman integration
+# @api private
+class puppet::server::foreman (
+  Boolean $katello = false,
+) {
+  if $katello {
+    include certs::puppet
```

Comment:
I assume we'll have this as a soft dependency where we assume it's presence if someone is setting katello true that they will have the requisite dependencies present? I am fine with that, and it helps make things easier, I am wanting to ensure we call that out (maybe in the Readme?).

Reply:
How about this?